generators/app/templates/server.ts

```diff
@@ -5,15 +5,15 @@ import { ApolloServer } from 'apollo-server-express';
 import * as Lodash from 'lodash';
 import { makeExecutableSchema } from 'apollo-server';
 import { mergeTypes } from 'merge-graphql-schemas';
-import * as dotenv from 'dotenv'
+import * as dotenv from 'dotenv';
 import jsonwebtoken from 'jsonwebtoken';
 
 import user from './graphql/schemas/user.graphql';
 import userResolver from './graphql/resolvers/userResolver';
 
 dotenv.config();
 
-Mongoose.connect('mongodb://'+process.env.DB_HOST+':'+process.env.DB_PORT, {
+Mongoose.connect('mongodb://' + process.env.DB_HOST + ':' + process.env.DB_PORT, {
```

Comment:
you could probably just do: 

`mongodb://${process.env.DB_HOST}:${process.env.DB_PORT}` with back-ticks instead of '

Reply:
yeah, i like this